processor/metricstransformprocessor/metrics_transform_processor.go

```diff
@@ -23,21 +23,37 @@ import (
 	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.opentelemetry.io/collector/consumer/pdatautil"
+	"go.uber.org/zap"
 )
 
+type mtpTransform struct {
```

Comment:
Can we just call this `transform` or maybe `transformCfg`? (same for `mtpOperation` below)

Reply:
I really want to make a separation between this config and the one in the config.go. imo mtpTransform makes it sound more like a mtp internal transform struct. Maybe call it internalTransform?

Reply:
Yea that sounds good to me